resources/assets/sass/app.scss

```diff
@@ -108,6 +108,18 @@ img {
   }
 }
 
+.checkbox-label {
+    display: block;
+    font-size: .75em;
```

Comment:
Could you put a `0` here--so `0.75em`? Also on lines 115 and 118. I just think CSS values read a little more clearly with the preceding 0.

Reply:
You're right. Prettier usually does that for me, but I disabled it so it wouldn't reformat the whole file. Thanks for the suggestion!